### PR TITLE
made necessary changes

### PR DESCRIPTION
**Pull request by Otakuwind**
Added

Proper links in a table
Getting started with Outreachy tasks
Open source contributions tasks
 Research section
 Recent learning blog post
 Accessibility blog post
 technical writing blog post

### CODE REVIEW

File: 2023/Nwajei Edgar/README.md [hunk 1]
```diff
@@ -1,19 +1,92 @@
...
-- [LinkedIn](https://www.linkedin.com/in/edgar-nwajei-5822631b7)
+
+## Links
+
+| Platform         | Link                                         |
+|------------------|----------------------------------------------|
+| GitHub Profile   | [https://github.com/Otakuwind](https://github.com/Otakuwind)       |
+| Personal Website | [https://0xwind.hashnode.dev](https://0xwind.hashnode.dev)     |
+| Twitter          | [https://twitter.com/0x_wind](https://twitter.com/0x_wind)       |
+| LinkedIn         | [https://www.linkedin.com/in/edgar-nwajei-5822631b7](https://www.linkedin.com/in/edgar-nwajei-5822631b7) |
```
Comment (thibaudcolas) [inline 2023/Nwajei Edgar/README.md:11]: It’s not a really good experience for anyone if you use link URLs as text.

File: 2023/Nwajei Edgar/README.md [hunk 2]
```diff
@@ -1,19 +1,92 @@
...
+
+
+
 ## Research
 - [Wagtail user guide](https://guide.wagtail.org/en-latest/)
 - [Official Wagtail website](https://wagtail.org/)
+
+## Recent learning.
+
+Short blog post on what i learnt recently https://0xwind.hashnode.dev/my-tech-trinity
```
Comment (thibaudcolas) [inline 2023/Nwajei Edgar/README.md:85]: Please make sure to add alt text to your images in the future, this is a fundamental web accessibility requirement.

File: 2023/Nwajei Edgar/README.md [hunk 3]
```diff
@@ -1,19 +1,92 @@
...
+
+### Accessibility features documentation
+Short blog post on accessibility: https://0xwind.hashnode.dev/enhancing-user-experience
```
Comment (thibaudcolas) [inline 2023/Nwajei Edgar/README.md:88]: This is a very generic article, please re-read the instructions:

> Demonstrate your understanding _based on past experiences_

File: 2023/Nwajei Edgar/README.md [hunk 4]
```diff
@@ -1,19 +1,92 @@
...
+
+
+## technical writing.
+Short blog post on technical writing : https://0xwind.hashnode.dev/my-javascript-journey
```
Comment (thibaudcolas) [inline 2023/Nwajei Edgar/README.md:92]: Again a very generic article. I want to read things that demonstrate your understanding based on your experience, not easily-googleable information.

File: 2023/Nwajei Edgar/README.md (hunk 2 above)
Comment (Otakuwind) [reply 2023/Nwajei Edgar/README.md:85]: i have noted all my mistakes, and will ensure I correct them properly to avoid repeating it in the future.

**Pull request merged**